Datahub.Core/Services/EmailNotificationService.cs

```diff
@@ -465,6 +466,50 @@ public string BuildAppLink(string contextUrl)
             ub.Path = contextUrl;
             return ub.ToString();
         }
+
+        public async Task SendFileSharingApprovalRequest(string username, string filename, DatahubProjectInfo projectInfo, IList<string> recipients)
+        {
+            var subject = "[DataHub] Public file sharing request";
```

Comment:
where is this localized? or is this something you will do later?

Reply:
Similarly to the other email notifications, it would have both English and French in the subject as well as the template. E.g.
`var subject = $"[DataHub] {serviceName} service created / {serviceName} service créé ";`